oskarstark/no-floc-listener/0.1/config/packages/oskarstark_symfony_no_floc_listener.yaml

```diff
@@ -0,0 +1,4 @@
+services:
+    OskarStark\Symfony\EventListener\NoFlocListener:
```

Comment:
I dont understand. 

This is an `EventSubscriberInterface`, why do you tag it as an `event_listener`?

It would be enough to remove the tag an add `autowire: true`

Reply:
Indeed, I mixed sth. up while copy pasting... 🤔